tests/php/test-amp-vimeo-embed.php

```diff
@@ -109,4 +118,23 @@ public function test__get_scripts( $source, $expected ) {
 
 		$this->assertEquals( $expected, $scripts );
 	}
+
+	/**
+	 * Test is_amp_shortcode_available_in_jetpack.
+	 *
+	 * @covers AMP_Base_Embed_Handler::is_amp_shortcode_available_in_jetpack()
+	 */
+	public function test_is_amp_shortcode_available_in_jetpack() {
```

Comment:
This test may not be necessary. I think we can rather just check if the `Jetpack_AMP_Vimeo_Shortcode` class exists.